src/test/java/MockTest.java

```diff
@@ -0,0 +1,435 @@
+import com.sun.javaws.exceptions.InvalidArgumentException;
```

Comment:
1. Czemu testy nie mają zdefiniowanego pakietu?
2. Wszystkie testy są w jednym pliku (nie liczę HttpRequestTest i ApplicationTest) przez co trudno się je czyta. Jak można by je podzielić?
3. Nie widzę testu, który sprawdzałby czy poprawnie obliczamy opłatę dla kierowcy.

Reply:
1. Nie mogłem się zdecydować jak je podzielić, ale myślę, że zdefiniowanie pakietów Testy integracyjne i testy jednostkowe mogłyby być jakimś startem.
2. No, to prawda. Ponownie nie doszedłem do żadnej konkluzji jak podzielić te testy. Myślałem, żeby podzielić je np. pod względem testowanych komponentów, albo testowanej funkcjonalności. Jakaś rada/opinia?
3. Są testy, które sprawdzają, czy otrzymywana jest właściwa odpowiedź przy zapytaniu o zebrane zarobki danego dnia, ale faktycznie nie ma konkretnych testów sprawdzających czy zwracana wartość przez PLNPricing jest właściwa. Jednak ciężko napisać taki test będąc świadomym, że sposób obliczania opłaty, oraz wartości za konkretne godziny mogą ulec zmianie. Czy w takim razie trzeba napisać mniej uniwersalne testy, które potencjalnie trzeba będzie usunąć/zmienić przy zmianach cen/sposobów obliczania?